static/js/admin/vue-permissions-editor/src/components/patronRoles.vue

```diff
@@ -23,15 +23,15 @@
         <ul class="set-patron-roles">
             <li v-if="isBulkMode">
                 <input type="radio" v-model="user_type" value="ignore"
-                       id="user_type_ignore"> <label for="user_type_ignore"> No Change</label>
+                       id="user_type_ignore"> <label for="user_type_ignore" @click="setRoleFromLabel('ignore')"> No Change</label>
```

Comment:
why is this near click binding and method needed now? Is it replacing any existing functionality?